style/style.scss

```diff
@@ -0,0 +1,67 @@
+$colortitulos: red;
+$colorsubtitulos: firebrick;
+$colornombres: brown;
+$background: cornflowerblue;
+$fuente: 20px ;
+$fuentefamilia: "Times New Roman", Times, serif;
+$imagenes: 270px;
+
+
+@mixin flexbox {
+    display:flex;
+    justify-content:start;
+}
+@mixin imagen {
+    display: flex;
+    justify-content: end;
+    
+}
+@mixin div {
+    display: flex;
+    
+}
```

Comment:
Bad use of `@mixins`

These are meant to be used as a single block of code that can be used everywhere, but repeating it like this defeats the purpose of it.

It's best that you use a mixin with variables so you can adapt it to every use-case you have. For instance:

```
@mixin flex($justify: center, $align: center, $direction: row){
    align-items: $align;
    display: flex;
    flex-direction: $direction;
    justify-content: $justify;
}
```

This means that you can use `@include flex()` and it will default every variable to its default (`align-items: center, justify-content: center & flex-direction: row`) but you can replace those defaults by passing the value you want in the order that they are assigned, for instance:

`@include flex(flex-start, flex-end, row)` means 

```
{
    align-items: flex-end;
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
}
```

For more information, you can read the documentation: https://sass-lang.com/documentation/at-rules/mixin